plugins/fireeye_hx/plugin.spec.yaml

```
@@ -368,3 +446,51 @@ actions:
         description: Alerts
         type: "[]alert"
         required: false
+  check_host_quarantine_status:
+    title: Check Host Quarantine Status
+    description: Check whether a host is quarantined or not
+    input:
+      agent_id:
+        title: Agent ID
+        description: The ID of the agent you want to check
+        type: string
+        required: false
+        example: 44d88612fea8a8f36de82e
+    output:
+      results:
+        title: Results
+        description: Results obtained for a specific agent
+        type: data
+        required: false
+  quarantine_host:
+    title: Quarantine Host
+    description: Quarantine a host
+    input:
+      agent_id:
+        title: Agent ID
+        description: The ID of the agent you want to isolate
+        type: string
+        required: false
+        example: 44d88612fea8a8f36de82e
+    output:
+      success:
+        title: Success
+        description: Whether the action was successful
+        type: boolean
+        required: false
```

```suggestion
        required: true
```
Shouldn't this always come back, as it can only indicate success or failure?